### DISCUSSION TITLE
Hera versions lower than 5 missing on PyPi

### DISCUSSION THREAD

**Discussion by janowskijak** [argoproj-labs/hera#546] [Q&A] [answered]
Hi,

Versions of Hera lower than 5 are missing on PyPi and can no longer be installed. Was this done on purpose?

**Comment by sambhav** [comment, marked as answer]
@janowskijak you can find older versions of hera at `hera-workflows`. See https://github.com/argoproj-labs/hera/discussions/532

**Comment by janowskijak** [comment]
Thanks @samj1912, did not realize this. Maybe a mention in readme would be helpful.

**Reply by flaviuvadan** [reply]
@janowskijak added a note in [this PR](https://github.com/argoproj-labs/hera/pull/549)

**Reply by janowskijak** [reply]
Awesome, much appreciated.
